### ISSUE TITLE
Update Benefits Pipeline

### ISSUE THREAD

**Issue by o-ram** [cal-itp/customer-success#383]
The Benefits pipeline in Hubspot currently serves as a glorified interest list, but going into 2024, we would like to have the pipeline have more detail, especially with regards to agencies who are "interested" vs "ready for onboarding". Therefore, the stages in the pipeline should be updated. While new properties may also add a helpful level of detail, the scope of this issue is just for stage updates at this time.

Current stages:
Interested, Agency Onboarding, Development, Testing, Launched

Proposed stages:
Interested, Ready for Onboarding, Actively Onboarding, Development, Testing, Launched

In these proposed stages, "interested" would refer to agencies who have mentioned being interested in some format but we don't know how serious the interest is and/or they do not have open loop payments yet. "Ready for onboarding" would denote agencies who have/are close to launching open loop payments and are actually ready to move forward with Benefits.

As part of updating the Benefits pipeline, the following items will take place. Currently being documented here but likely to be broken out in separate issues for each task.

- [x] Change stages
~~[ ] Implement usage of "priorities" _(potential to be broken out into a separate issue)_ There are still no current plans to start using the priority feature in Hubspot, so this will not be implemented.~~
- [x] Create workflows tied to when the onboarding form should be filled out _(anything related to form answers is not in scope for this issue)_. Form creation is documented in #388 
- [x] Draft a pipeline guide for the Benefits pipeline _(**in progress**)_
- [x] Create workflows to automatically create a deal in the Benefits pipeline once it reaches a certain stage in the payments pipeline
- [x] Create a tag for "benefits prospect"

**Comment by AnthonyRollins** [comment]
@o-ram Is this something we are thinking needs to be done before February?

**Comment by o-ram** [comment]
@AnthonyRollins ideally yes, but I haven't been able to get to it. Slotted it in for next week.

**Comment by o-ram** [comment]
Benefits pipeline is fully updated as of 8/12: all stages have been renamed, workflows have been activated and tested multiple times, and all _active_ deals are updated with the latest information. A pipeline guide has also been drafted and was passed to @laneymangan to review from the CS side with a fresh pair of eyes. Once Laney has reviewed it, I will go over it with @indexing.
